dist/assets/css/footer.css

```diff
@@ -0,0 +1,3 @@
+.footer {
+  color: #f1feff;
```

Comment:
consider using existing global tailwind colors, or add this color to the global if it does not exist and then use it.

Reply:
Done